packages/browser/src/integrations/globalhandlers.ts

```diff
@@ -104,14 +105,6 @@ export class GlobalHandlers implements Integration {
 
     const newEvent: Event = {
```

Comment:
We can use just `event`, as it has no new data added (just remove this line and change `addEx` call and return `event`